app/src/main/kotlin/com/simplemobiletools/dialer/fragments/RecentsFragment.kt

```diff
@@ -150,6 +162,39 @@ class RecentsFragment(context: Context, attributeSet: AttributeSet) : MyViewPage
         }.toMutableList() as ArrayList<RecentCall>
 
         recents_placeholder.beVisibleIf(recentCalls.isEmpty())
-        (recents_list.adapter as? RecentCallsAdapter)?.updateItems(recentCalls, text)
+        recentsAdapter?.updateItems(recentCalls, text)
+    }
+}
+
+// hide private contacts from recent calls
```

Comment:
this is another issue, we should show these items, without being clickable, with some custom label as in other apps. Do you have the option to reproduce private calls?

Reply:
I've made it the same way it was before my commit. Do you mean that these recents for private contacts should be shown in the list, but be non-clickable and with some custom label? What the label should be then?

Reply:
ye I know that it was like that previously, just thinking of some improvements. Maybe we can call it "Unknown caller" or so. Not sure how is it called in other apps, I cannot really reproduce those calls intentionally.

Reply:
> this is another issue, we should show these items, without being clickable, with some custom label as in other apps. Do you have the option to reproduce private calls?

Can you also specify other apps where can I see the right implementation? Thanks.

Reply:
> ye I know that it was like that previously, just thinking of some improvements. Maybe we can call it "Unknown caller" or so. Not sure how is it called in other apps, I cannot really reproduce those calls intentionally.

Gotcha! I'll implement it. I'll do it in another branch and pull request.